packages/design-system-react/src/components/icon/.svgrrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
This configuration file controls how SVGR transforms our SVG files into React components. It:
- Enables TypeScript output
- Adds React ref forwarding for component composition
- Sets consistent SVG props (role="img" and fill="currentColor")
- Uses our custom template for consistent component structure
- Configures SVGO optimization settings to preserve viewBox and ensure consistent color handling

This ensures all our icon components are generated with consistent typing, accessibility, and styling capabilities.